docs/reference/Func/NIFReadEntriesExt.md

```diff
@@ -139,5 +154,6 @@ Info we want. */
 	 OSMemFree(hBuffer);
 }
 ```
+
```

Comment:
Can some "see also" be added here if possible. like API related to "NIF" . Check other NIF related API documentation for reference.

Reply:
Added.